azurerm/internal/services/media/media_services_account_resource.go

```diff
@@ -97,7 +97,8 @@ func resourceMediaServicesAccount() *schema.Resource {
 							Optional:         true,
 							DiffSuppressFunc: suppress.CaseDifference,
 							ValidateFunc: validation.StringInSlice([]string{
-								"SystemAssigned",
+								string(media.ManagedIdentityTypeSystemAssigned),
+								string(media.ManagedIdentityTypeNone),
```

Comment:
rather than adding `None` as an option here, Terraform can use the omission of this block to indicate that this should be `None` instead, so can we remove this?

```suggestion
```

Presumably this means we need to mark the parent block as not computed too: https://github.com/terraform-providers/terraform-provider-azurerm/pull/10133/files#diff-bf7abb9831a61a65d7d6d830814a15135018ee57b20642035008f747ef86968fR81